src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1068,16 +1068,20 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         try {
           checkoutCommand.execute();
-        } catch(GitLockFailedException e) {
+        } catch (GitLockFailedException e) {
```

Comment:
...if you expect me to fix all the demented formatting in the git plugin on
this pull request, you need to reconsider. =)